src/test_creation/modules/workflow/parse.py

```diff
@@ -27,3 +37,69 @@ def get_completeness_score(self, score_format: str = 'fraction', verbose: bool =
             print(f'Score: {score}')
             print()
         return score
+
+    # FIXME From checklist.py. To be refactored 
+    def _get_md_representation(self, content: dict, curr_level: int):
+        repeated_col = [k for k, v in content.items() if isinstance(v, list)]
+
+        # print out header for each item
+        md_repr = '#' * curr_level
+        if 'ID' in content.keys():
+            md_repr += f" {content['ID']}"
+        if 'Title' in content.keys():
+            md_repr += f" {content['Title']}\n\n"
+        elif 'Topic' in content.keys():
+            md_repr += f" {content['Topic']}\n\n"
+
+        # print out non-title, non-repeated items
+        for k, v in content.items():
+            if k not in repeated_col and k not in ['Title', 'Topic', 'ID']:
+                md_repr += f'**{k}**: {v.replace("'", "\\'")}\n\n'
+
+        # handle repeated columns and references
+        point_form_col = ['References', 'Function References', 'Observations']
+        for k in repeated_col:
+            if k not in point_form_col:
+                for item in content[k]:
+                    md_repr += self._get_md_representation(item, curr_level=curr_level + 1)
+            else:
+                md_repr += f'**{k}:**\n\n' + '\n'.join(f'  - {item}' for item in content[k]) + '\n\n'
+
+        return md_repr
+    
+    # FIXME. From checklist.py. To be refactored 
+    @staticmethod
+    def __filedump_check(output_path: str, exist_ok: bool):
+        if not exist_ok and os.path.exists(output_path):
+            raise FileExistsError("Output file already exists. Use `exist_ok=True` to overwrite.")
+        return True
+
+    # FIXME. From checklist.py. To be refactored 
+    def export_html(self, content: str, output_path: str, exist_ok: bool = False):
+        self.__filedump_check(output_path, exist_ok)
+        pypandoc.convert_text(content, 'html', format='md', outputfile=output_path)
+
+    # FIXME. From checklist.py. To be refactored 
+    def export_pdf(self, content: str, output_path: str, exist_ok: bool = False):
+        self.__filedump_check(output_path, exist_ok)
+        pypandoc.convert_text(content, 'pdf', format='md', outputfile=output_path,
+                              extra_args=['--pdf-engine=tectonic'])
```

Comment:
As discussed, I will take this part out and make it into a mixin so that both `Checklist` and `ResponseParser` (and any future objects that supports exporting HTMLs/PDFs) can use the same method without duplication.